### PR TITLE
Add Level Editor

### PR DESCRIPTION
**Pull request by devrajneupane**
This PR is initial implementation for Map/Layout Editor. currently it only place hard coded item on Map on click event

https://github.com/devrajneupane/Bomberman/assets/51816083/82bad499-c7f1-414a-b517-47784fc7429c

### CODE REVIEW

**Review by aman007shrestha — APPROVED** [review]

File: src/components/Layout.ts
Comment (aman007shrestha) [inline src/components/Layout.ts]: Keep this in constants. Why 15 instead of 16 here?

File: src/components/Layout.ts
Comment (aman007shrestha) [inline src/components/Layout.ts:55]: should not concrete wall always be there in odd fields ? If i is odd and j is odd automatically place concrete wall instead of always depending on json map input data ? Your call

**Review by aman007shrestha — APPROVED** [review]

File: src/components/Layout.ts
Comment (devrajneupane) [reply src/components/Layout.ts:55]: According to the game, yes. I wanted to provide flexibility to the level editor, allowing users to place any item wherever they prefer except in border concrete wall.

File: src/components/Layout.ts
Comment (devrajneupane) [reply src/components/Layout.ts]: yes, it should be 16. i will put them in constants before merging this PR

**Pull request merged**
